cot/src/body.rs

```diff
@@ -243,6 +243,26 @@ impl http_body::Body for Body {
     }
 }
 
+macro_rules! body_from_impl {
+    ($ty:ty) => {
+        impl From<$ty> for Body {
+            fn from(buf: $ty) -> Self {
+                Self::new(BodyInner::Fixed(Bytes::from(buf)))
+            }
+        }
+    };
+}
+
+body_from_impl!(&'static [u8]);
+// body_from_impl!(std::borrow::Cow<'static, [u8]>);
```

Comment:
Why this commented out code?

Reply:
Leftover from previous attempts I've forgotten to remove, thank you for catching that!